guidance_pipeline/main_test.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+)
+
+var update = flag.Bool("update", false, "update .golden files")
+
+func TestMain(t *testing.T) {
```

Comment:
[TestMain](https://golang.org/pkg/testing/#hdr-Main) is sort of reserved. Not sure what would be better.